### PR TITLE
Added developement scripts for gravity

### PR DESCRIPTION
**Pull request by ppande1020**
Checking in build and test powershell scripts to aid in CI

### CODE REVIEW

**Comment by mrobustelli** [comment]
The only comment I have from the review is the naming of the default.ps1 script.  It really doesn't tell me what the script does.

File: developmentScripts/default.ps1 [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+task TestInitialize -Description "Cleanup pre-existing test artifacts, if they exist" {
+    InitializeDirectory $test_logs
+}
+
+task Compile -Depends CompileInitialize, NuGetRestore -Description "Compile the solution" {
+    Write-Verbose "Configuration: $build_config"
+    Write-Verbose "Verbosity: $verbosity"
+
+    # https://msdn.microsoft.com/en-us/library/ms164311.aspx
+    exec { msbuild @($solution,
```
Comment (mrobustelli) [inline developmentScripts/default.ps1]: This throws an error if using the VS2015 MSBuild.  Looks like Psake uses a setting in the registry HKEY_LOCAL_MACHINE -> SOFTWARE -->Microsoft --> MSBuild -->ToolsVersion --> 14.0.  I updated the MSBuildToolsPath to C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\Bin\ and it works.  I will create an issue for this, but I think we should update the error message if possible to say that changing this key may get it working.

Comment (ppande1020) [reply developmentScripts/default.ps1]: I have added this bit to the script so a user knows to update the MSBUILD version in the registry key. I also found that installing a lower version of nuget makes psake pick MSBUILD for VS 2015. I have added this bit to the build.ps1 script

**Comment by ppande1020** [comment]
I have renamed default.ps1 to defaultBuildTest.ps1

**Pull request merged**